floodscan_susceptibliity_target_pipeline.R

```diff
@@ -152,33 +162,64 @@ list(
   ### Admin 2 - method 1: binary reclassification ####
   
   # need to separate Mozambique from the rest as running into memory issues
+  ## Doing this by country and season
+  #tar_target(
+  #  name = df_adm2_stats_binary_test,
+  #  command = c("Kenya", "Ethiopia", "Somalia", "Mozambique") %>%
+  #    map_df(
+  #      \(country_name){
+  #        zonal_pop_exposure(
+  #          floodscan_path = fp_fs,
+  #          worldpop_inputs = keep_at(lr_wp, country_name),
+  #          flood_frac_thresh = 0.2,
+  #          binarize_floodscan = T,
+  #          adm = lgdf_adm$adm2 %>% 
+  #            filter(adm0_en == country_name),
+  #          country_seasons = pluck(ssn_by_country, country_name),
+  #          cols_keep = c("adm0_en", "adm1_en", "adm1_pcode", "adm2_en", "adm2_pcode")
+  #        )
+  #        print(paste0("Completed processing for: ", country_name))
+  #      }
+  #    ) %>%
+  #    bind_rows()
+  #),
+  
   tar_target(
-    name = df_ken_eth_som_adm2_stats_binary,
+    name = df_eth_som_adm2_stats_binary,
     command = zonal_pop_exposure(
       floodscan_path = fp_fs,
-      worldpop_inputs = discard_at(lr_wp,"Mozambique"),
-      flood_frac_thresh = 0.2,
-      binarize_floodscan = T,
+      worldpop_inputs = discard_at(lr_wp, c("Kenya", "Mozambique")),
+      flood_frac_thresh = c(0.005, 0.01, 0.05, 0.1, 0.15, 0.2),
+      binarize_floodscan = F,
```

Comment:
you realize that binary is set to `FALSE`, but you are calling the target `*_stats_binary`.

I think you should clear this up in the function.